core/ir/instr_api.h

```diff
@@ -1928,6 +1928,16 @@ DR_API
  *   non-NULL, the type of the floating point operation is written to \p type.
  */
 bool
+instr_is_floating_exten(instr_t *instr, dr_instr_category_t *type);
```

Comment:
Not sure about the name "_exten": how about `instr_is_floating_type` or `instr_get_floating_type`

Reply:
instr_is_floating_type